web/web.go

```diff
@@ -146,9 +146,13 @@ func New(st local.Storage, qe *promql.Engine, rm *rules.Manager, status *Prometh
 	instrf := prometheus.InstrumentHandlerFunc
 	instrh := prometheus.InstrumentHandler
 
-	router.Get("/", instrf("status", h.status))
+	router.Get("/", instrf("graph", h.graph))
```

Comment:
> Querying is the primary use case for a timeseries database

Yes, but not directly via /graph. /graph is for when your dashboards can't help you, or you're debugging exactly what's going on with a timeseries/rule/alert.

The status page needs improvement, but that doesn't mean we should replace it with /graph.